### PR TITLE
font-variant tests

### PR DESCRIPTION
**Pull request by svgeesus**
4 font-variant tests, with refs. @litherum for review, tests staged at https://www.w3.org/People/chris/fwf/index.html

### CODE REVIEW

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/230823466)

# Firefox (nightly) #
Testing web-platform-tests at revision d75ca015af9b397da9622ac8c5ae3071d744fe6a
Using browser at version BuildID 20170509100208; SourceStamp b21b974d60d3075ae24f6fb1bae75d0f122f28fc
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>4 tests ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/5874/css/css-fonts-3/font-variant-01.html">/css/css-fonts-3/font-variant-01.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5874/css/css-fonts-3/font-variant-02.html">/css/css-fonts-3/font-variant-02.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5874/css/css-fonts-3/font-variant-03.html">/css/css-fonts-3/font-variant-03.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5874/css/css-fonts-3/font-variant-04.html">/css/css-fonts-3/font-variant-04.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
</details>

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/230823472)

# Chrome (unstable) #

File: css/css-fonts-3/font-variant-03.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+<title>CSS Test:  font-variant: normal; low level equivalence</title>
+<link rel="author" title="Chris Lilley" href="chris@w3.org">
+<link rel="help" href="https://www.w3.org/TR/css-fonts-3/#font-variant-prop">
+<link rel="help" href="https://www.w3.org/TR/css-fonts-3/#font-feature-settings-prop">
+<link rel="match" href="font-variant-03-ref.html">
+<meta name="assert" content="The value ‘normal’ resets all subproperties of ‘font-variant’ to their inital value. It does not reset the values of either ‘font-language-override’ or ‘font-feature-settings’">
+<style>
+  @font-face {
+    font-family: fwf;
+    src: url(support/fonts/FontWithFeatures2.otf);
```
Comment (litherum) [inline css/css-fonts-3/font-variant-03.html]: Are you sure this is the right font? https://github.com/w3c/web-platform-tests/pull/5875/files uses a different one.

**Comment by litherum** [comment]
r=me

**Pull request merged**